src/SeaNetMicron.cpp

```diff
@@ -89,43 +98,35 @@ void Micron::configure(const MicronConfig &config,uint32_t timeout)
     //some conversions
     int ad_interval = (((config.resolution/config.speed_of_sound)*2.0)*1e9)/640.0;
     int number_of_bins = config.max_distance/config.resolution;
-    uint16_t left_limit = (((M_PI-config.left_limit.rad)/(M_PI*2.0))*6399.0);
-    uint16_t right_limit = (((M_PI-config.right_limit.rad)/(M_PI*2.0))*6399.0);
+    uint16_t left_limit = (((M_PI+config.left_limit.rad)/(M_PI*2.0))*6399.0);
+    uint16_t right_limit = (((M_PI+config.right_limit.rad)/(M_PI*2.0))*6399.0);
```

Comment:
I guess you are working with a different axes convention. In most parts of rock we have the right hand rule, where x is forward, y is left and z is up. In this case you would express a 90° window to the front on a horizontal plane with a positive left angle of 45° and a negative right angle of -45°. Since the Micron internally goes from 0-6399 from the right to the left, the conversion should be correct. But there might be definitely a lack of documentation.